docs/guide/installation.md

```diff
@@ -21,11 +21,22 @@ npm install remirror@next @remirror/pm@next
 You will also notice that you are also installing `@remirror/pm`. This is a peer dependency to all the scoped `@remirror` packages which interact with `prosemirror-*`. It provides some assurances around the versions of prosemirror libraries being used and helps simplify the codebase.
 
 ## Browser support
+Remirror aims to be compatible with all browsers released since **2017**. For this it's required to included the remirror styles.
+```Javascript
```

Comment:
I think it's best to move this section to a new block **Required styles**. It's not really a new or old browser issue. It's more that there are styles which Firefox, and even Chrome require. 

Without the styles on Chrome, whitespace doesn't get added properly. So, you can't insert a <kbd>Space</kbd> character at the end of the document. 

I guess the summary is that it fixes a bunch of browser inconsistencies and is required.